core/storage/statistics/gae_models.py

```diff
@@ -615,6 +615,73 @@ def create(
         return entity_id
 
 
+class ExplorationStatsModel(base_models.BaseModel):
+    """Model for storing analytics data for an exploration.
+
+    The ID of instances of this class has the form {{exp_id}}.{{exp_version}}.
+    """
+    # ID of exploration.
+    exp_id = ndb.StringProperty(indexed=True)
+    # Version of exploration.
+    exp_version = ndb.IntegerProperty(indexed=True)
+    # Number of students who attempted the exploration.
```

Comment:
Might want to qualify "actual" here -- i.e. explain clearly what this represents.

Reply:
Done.